projects/ngworker/lumberjack/http-driver/package.json

```
@@ -0,0 +1,10 @@
+{
```

This configuration needs to be converted into a *sub-module* configuration like [this sample](https://github.com/ng-packagr/ng-packagr/blob/master/integration/samples/secondary/sub-module/package.json).

```suggestion
{
  "name": "@ngworker/lumberjack/http-driver",
  "private": true,
```

I don't know if we *have* to add a `"version"` declaration.

---

From reading the docs, I would believe it automatically keeps the version in sync; I don't think we need to add it.